Add communications editor job description (#183)
publiccodenet/publiccode.net: careers/content-editor.md

Part of this effort is to invite people to join us. This means:

* telling the story of public code so that it's exciting and compelling for both our contributors, and the broader government and technology communities
* making sure our information and documentation is clear, approachable and accessible to anyone who wants to use our materials, work with us, or contribute to make them better

In the coming year, we want to significantly increase awareness of the concept of public code and the results we're already seeing in the codebases we work with.

## What the Communications editor does

As a Communications editor, you work together with the community to highlight our impact and enable others to participate. This means you will:

* plan and create informational content, blogposts, social media posts, creative campaigns, presentations, and possibly print and film

> * help colleagues and community members share accomplishments and learnings from their work (including interviewing, editorial direction, narrative construction, encouragement)

Reviewer: ```suggestion
* help colleagues and community members share accomplishments and learnings from their work (including interviewing, editorial direction, narrative construction, and encouragement)
```

---
type: Position
excerpt:  Tell stories about public code's impact that everyone can understand
---

# Communications editor/storyteller

Come help us enable public-purpose software and policy that is open and collaborative. We support open source codebases developed by public organizations to become thriving, community-driven projects that are sustainable and scalable. By doing so, we help public organizations take back control over their technology and develop more effective public services with less risk.

Part of this effort is to invite people to join us. This means:

* telling the story of public code so that it's exciting and compelling for both our contributors, and the broader government and technology communities
* making sure our information and documentation is clear, approachable and accessible to anyone who wants to use our materials, work with us, or contribute to make them better

> In the coming year, we want to significantly increase awareness of the concept of public code and the results we're already seeing in the codebases we work with.

Reviewer 2: Suggested simplification, not necessary

```suggestion
In the coming year, we want to significantly increase awareness of what public code is and the results we're already seeing in the codebases we work with.
```

## What the Communications editor does

As a Communications editor, you work together with the community to highlight our impact and enable others to participate. This means you will:

* plan and create informational content, blogposts, social media posts, creative campaigns, presentations, and possibly print and film
* help colleagues and community members share accomplishments and learnings from their work (including interviewing, editorial direction, narrative construction, and encouragement)
* work with people across the organization and our broader member and codebase community, as well as external designers, animators and other creatives
* strategically use a wide range of platforms and channels to share our work (potentially including podcasts, videos, and social media)
* find opportunities in others' events, podcasts or other activities for our colleagues to contribute
* support colleagues working with press and media
* apply a user-centered content design approach to content marketing

> * proofread colleagues' pull requests on Github for style, usefulness and formatting (hyperlinks and Markdown)

Author: ```suggestion
* proofread international colleagues' pull requests on Github for accessible international English, plus style, usefulness and formatting (hyperlinks and Markdown)
```

* use analytics to monitor successful content

If you enjoy combining your storytelling and analytical skills to make complex content accessible and compelling, then we'd love to hear from you.

## What we expect from all of our team members

You should:

* share our core values: openness, quality, trust, community, helpfulness
* understand the value of and be experienced with open source products, git and communities
* want to work for the public good
* have a collaborative working attitude and take pride in making things together
* be able to work independently
* have international experience

## What we expect from the Communications editor

We expect our new Communications editor to:

* be diplomatic, strategically aware and able to be a strong advocate for the organization
* have proven ability to write about complicated things in a simple way for a wide audience
* be able to identify good stories that will matter to our audience
* contribute to our awareness raising strategy as the organization moves from a start up to scaling phase

> * have native level English fluency

Reviewer 2: I don't think 'native' is the level we're looking for, since it is mostly about cultural binding. Native speakers often frustrate inter-cultural English conversations by not being able to bridge this gap.

For example: we might want someone with a strong French accent who is adept at cross-cultural communication over a native from Australia that likes their sayings.

```suggestion
* able to communicate in English fluently in an international context
```

Reviewer 2: What about adding proofread to the requirement?

```suggestion
* able to edit and proofread English in an international context
```

You could add 'writing' in there to clarify, and we could add 'and intercultural' to the context.

> * have native level English fluency

Author: ```suggestion
* be a strong communicator in written and spoken English
```
